docs/guides/map-data/index.md

```diff
@@ -0,0 +1,87 @@
+# Map Data
+
+## General Information
+
+Map/Geography data is available in the SDE or (partially) through ESI.  
```

Comment:
Geography is the study of the Earth. Could say "Map data", or "Map/Universe data", or "Map/Cluster data".